src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx

```diff
@@ -78,6 +75,7 @@ const getText = (
 
   return {
     title: "Start curating your list of works",
-    description: "Add works from All Saves or add new artworks as you browse.",
+    description:
+      "Add works from Saved Artworks or add new artworks as you browse.",
```

Comment:
**suggestion**: Can we keep some of these strings in the newish [I18n translations file](https://github.com/artsy/force/blob/main/src/System/i18n/locales/en-US/translation.json)?

Reply:
we can do this, but we will need to make quick refactor. do you mind if I do this in a separate PR?